src/Microsoft.DotNet.Build.Tasks/PackageFiles/codeOptimization.targets

```diff
@@ -6,8 +6,40 @@
     <OptimizationDataVersion Condition="'$(OptimizationDataVersion)'==''">2.0.0-rc-61101-16</OptimizationDataVersion>
     <OptimizationDataPackageName Condition="'$(OptimizationDataPackageName)'==''">RoslynDependencies.OptimizationData</OptimizationDataPackageName>
     <OptimizationDataDir Condition="'$(OptimizationDataDir)'==''">$(ToolsDir)OptimizationData/</OptimizationDataDir>
+    <RestoreDefaultOptimizationDataPackage Condition="'$(RestoreDefaultOptimizationDataPackage)'==''">true</RestoreDefaultOptimizationDataPackage>
+    <UsePartialNGENOptimization Condition="'$(UsePartialNGENOptimization)'==''">true</UsePartialNGENOptimization>
   </PropertyGroup>
 
+  <!-- If IBC data hasn't been merged with the IL yet, preprocess it first -->
+  <Target Name="PreProcessIBCData"
+          BeforeTargets="OptimizeWithTrainingData"
```

Comment:
Possibly... the condition isn't actually identical. `OptimizeWithTrainingData` looks for counts in `*.pgo` files. This step is conditional on `*.dll` and generates `*.pgo` (which would otherwise be missing). Would this execute before `OptimizeWithTrainingData`'s condition check if the latter depends on this target?